docs/concepts/composition.md

```diff
@@ -448,6 +454,16 @@ spec:
         - type: math
           math:
             multiply: 1024
+    # Patches can also be applied from the composed resource (MySQLServer)
```

Comment:
Do they also appear in claim status?

Reply:
Yes, `ToCompositeFieldPath` patches are authoritative for status fields, and can also late initialize spec fields. This applies to both Composites and Claims.

Reply:
Updated comment to clarify that composite is patched and if a claim reference is available, then the claim is also patched.